index.js

```diff
@@ -137,21 +150,68 @@ function useCard()
     //Play card if valid move, otherwise ignore
     if (isValidCard == true)
     {
+        let cardBeingPlayed = players[gameTurn].playerDeck.getCard(cardIndex);
         alert("Debug: Valid move.");
+        
+        
+        //Will run if there is a stackable card played, +2 or +4
+        if(drawStack.stackAmt != 0){
+                if(cardBeingPlayed.value != drawStack.cardValue){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }else if(cardBeingPlayed.value == 1 && cardBeingPlayed.color != 'Special'){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }
+                
+        }
+        
         players[gameTurn].playerDeck.playCard(cardIndex);
-        gameTurn++;
+        
+        if(cardBeingPlayed.color == 'Special'){
```

Comment:
If player chooses a special 0 or 1 card, and leaves the uno tab before picking a color, then upon returning to the uno game the popup to choose color is gone and there is no color set for the current round, just shows -1 for the card value.

Reply:
Could not recreate issue, leaving tab and returning works on my end.